pkg/flink/lsp/lsp_completer_ws.go

```diff
@@ -77,10 +77,10 @@ func (w *WebsocketLSPClient) refreshWebsocketConnection() {
 	}
 }
 
-func NewWebsocketClient(getAuthToken func() string, baseUrl, organizationId, environmentId string, handlerCh chan *jsonrpc2.Request) LspInterface {
+func NewWebsocketClient(getAuthToken func() string, baseUrl, organizationId, environmentId string, handlerCh chan *jsonrpc2.Request) (LspInterface, error) {
```

Comment:
Interesting that this is part of this PR's diff

Reply:
Oh, that's because I moved two commits at once; I think it was the commit right before the flink update one. Or maybe the one right after.